<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dg_AddProject</class>
 <widget class="QDialog" name="Dg_AddProject">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="addproject" native="true">
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>850</x>
        <y>11</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/new/home/home/guanbi.png);
border-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="pic">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>900</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"> 

</string>
      </property>
      <property name="text">
       <string>  新增支持测试卡</string>
      </property>
     </widget>
     <widget class="QFrame" name="frame_addpro">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>59</y>
        <width>881</width>
        <height>431</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>0</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>测试卡信息</string>
       </property>
      </widget>
      <widget class="QFrame" name="frame">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>361</width>
         <height>391</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>61</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>卡号ID:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="ed_id">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>10</y>
          <width>271</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
       <widget class="QLineEdit" name="ed_name">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>50</y>
          <width>271</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>61</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>卡名称:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>100</y>
          <width>81</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>孵化时间：</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="ed_time">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>100</y>
          <width>81</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>100</y>
          <width>81</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>秒钟</string>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>150</y>
          <width>341</width>
          <height>181</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="title">
         <string>分析项目</string>
        </property>
        <widget class="ViceButton" name="bt_clear">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>70</y>
           <width>71</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border-image: url(:/new/home/home/button.png);</string>
         </property>
         <property name="text">
          <string>清空</string>
         </property>
        </widget>
        <widget class="QPlainTextEdit" name="edp_pro">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>30</y>
           <width>221</width>
           <height>141</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>350</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>T信号线条数</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="ed_t">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>350</y>
          <width>231</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </widget>
      <widget class="QFrame" name="frame_2">
       <property name="geometry">
        <rect>
         <x>510</x>
         <y>30</y>
         <width>361</width>
         <height>391</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QListView" name="lv_card_addpro">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>341</width>
          <height>321</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
       <widget class="ViceButton" name="bt_add">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>345</y>
          <width>341</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/new/home/home/button.png);</string>
        </property>
        <property name="text">
         <string>列表当中不存在，新增</string>
        </property>
       </widget>
      </widget>
      <widget class="ViceButton" name="bt_move">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>200</y>
         <width>71</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/new/home/home/button.png);</string>
       </property>
       <property name="text">
        <string>&lt;&lt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>510</x>
         <y>0</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>已有分析项目</string>
       </property>
      </widget>
     </widget>
     <widget class="MainButton" name="bt_ok">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>500</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
     <zorder>pic</zorder>
     <zorder>pushButton_2</zorder>
     <zorder>frame_addpro</zorder>
     <zorder>bt_ok</zorder>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MainButton</class>
   <extends>QPushButton</extends>
   <header location="global">mainbutton.h</header>
  </customwidget>
  <customwidget>
   <class>ViceButton</class>
   <extends>QPushButton</extends>
   <header location="global">vicebutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
